public/chart.js

```diff
@@ -111,6 +128,15 @@ function characterChart(svg, dataURL) {
         };
     }
 
+    function convertEpisodesCSV(d) {
+        // convert string data from CSV
+        return {
+            date: parseDate(d.date.substring(0, 10)),
```

Comment:
Fixed it server-side already. No need for the substring anymore (`.slice()` is actually what you are looking for `.substring()` and `.substr()` are both somehow broken)